### ISSUE TITLE
speed deviations for persons (trac #3517)

### ISSUE THREAD

**Issue by behrisch** [eclipse/sumo#3517]
- [x]  should be calculated only once for every person (not once for every walk)
- [x] should be respected by TraCI as well

Migrated from http://sumo.dlr.de/ticket/3517
```json
{
    "status": "new", 
    "changetime": "2017-10-20T16:24:47Z", 
    "description": "* should be calculated only once for every person (not once for every walk)\n* should be respected by TraCI as well", 
    "reporter": "behrisch", 
    "cc": "", 
    "resolution": "", 
    "_ts": "1508516687431583", 
    "component": "simulation data input", 
    "summary": "speed deviations for persons", 
    "priority": "major", 
    "keywords": "", 
    "time": "2017-10-18T13:27:12Z", 
    "milestone": "1.0.0", 
    "owner": "", 
    "type": "defect"
}
```

**Comment by namdre** [comment]
Speedfactor only exists once in the person.
The value is not yet accessible via TraCI

**Comment by namdre** [comment]
person.getSpeedFactor and setSpeedFactor were implemented in #8576
